GGProject/static/admin/css/changelists.css

```diff
@@ -0,0 +1,344 @@
+/* CHANGELISTS */
+
+#changelist {
+    position: relative;
+    width: 100%;
+}
+
+#changelist table {
+    width: 100%;
+}
+
+.change-list .hiddenfields { display:none; }
```

Comment:
Looks like most of the other css with one property is on multiple lines, we should be consistent